BREAKING.md

```diff
@@ -14,6 +14,16 @@ There are a few steps you can take to write a good change note and avoid needing
 - Provide guidance on how the change should be consumed if applicable, such as by specifying replacement APIs.
 - Consider providing code examples as part of guidance for non-trivial changes.
 
+# 2.0.0
```

Comment:
Can you please sync up with Curtis RE versions we will use post 1.0?
I'd think (based on [this doc](https://microsoft-my.sharepoint-df.com/:w:/r/personal/curtism_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B3AB9AB78-0F77-4D41-88FE-245FCB5BB7AD%7D&file=Fluid%20Framework%20Post-1.0%20Version%20Schema.docx&wdOrigin=OFFICECOM-WEB.MAIN.SEARCH&ct=1655355806346&action=default&mobileredirect=true&share=IQF4q7k6dw9BTYj-JF_LW7etAVY8M39Gt7tJksWfeMQeRBQ) it will be 2.0.0-alpha.1.0.0 where these changes will show up for internal customers.